conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/logging/LorreInfoLogging.scala

```diff
@@ -65,7 +65,7 @@ trait LorreInfoLogging {
       platform.name,
       platformConf.network,
       showPlatformConfiguration(platformConf),
-      lorreConf.headHash.fold("head")(_.value),
+      lorreConf.headHash.fold("head")(identity),
```

Comment:
getOrElse will read better than a fold with identity transform

Reply:
Where was I, when I made this change